submit-api/src/Cardano/TxSubmit/Node.hs

```diff
@@ -226,12 +240,12 @@ nullChainSyncWithBlocksPtcl =
 --
 txSubmissionClient
   :: TxSubmitVar -> TxSubmitMetrics
-  -> LocalTxSubmissionClient (GenTx ByronBlock) (ApplyTxErr ByronBlock) IO Void
+  -> LocalTxSubmissionClient (GenTx ByronBlock) (ApplyTxErr ByronBlock) IO ()
 txSubmissionClient tsv metrics =
     LocalTxSubmissionClient $
       readTxSubmit tsv >>= pure . loop
   where
-    loop :: GenTx ByronBlock -> LocalTxClientStIdle (GenTx ByronBlock) (ApplyTxErr ByronBlock) IO Void
+    loop :: GenTx ByronBlock -> LocalTxClientStIdle (GenTx ByronBlock) (ApplyTxErr ByronBlock) IO ()
```

Comment:
Why it was necessary to switch from `Void` to `()`?

Reply:
Right, I thinks it should be `Void`. I changed it